### PR TITLE
Redefine the auto_id and primary_key

### PR DESCRIPTION
**Pull request by DragonDriver**
Signed-off-by: dragondriver <jiquan.long@zilliz.com>

### CODE REVIEW

File: pymilvus/client/abstract.py [hunk 1]
```diff
@@ -254,6 +258,70 @@ def distances(self):
         return self._raw.scores
 
 
+"""
```
Comment (czs007) [inline pymilvus/client/abstract.py:261]: delete this, my fault.

File: pymilvus/client/asynch.py [hunk 2]
```diff
@@ -231,12 +231,36 @@ def on_response(self, response):
         return ChunkedQueryResult(response, self._auto_id)
 
 
+"""
```
Comment (czs007) [inline pymilvus/client/asynch.py:234]: delete this please

File: pymilvus/client/abstract.py [hunk 3]
```diff
@@ -107,7 +109,9 @@ def dict(self):
         _dict["description"] = self.description
         _dict["type"] = self.type
         _dict["params"] = self.params or dict()
-        _dict["is_primary"] = self.is_primary_key
+        _dict["is_primary"] = self.is_primary
+        _dict["is_primary_key"] = self.is_primary
```
Comment (czs007) [inline pymilvus/client/abstract.py:113]: redundant?

**Comment by DragonDriver** [comment]
PTAL @czs007

**Review by czs007 — APPROVED** [review]

**Pull request merged**